cloud/shared/bin/lib/config_validator.py

```diff
@@ -0,0 +1,36 @@
+import os
+import json
+
+class ConfigValidator(): 
+    def __init__(self, config_filename, variable_definition_filename):
+        with open(config_filename, 'r') as config_file:
+            self.configs = json.loads(config_file.read())
+
+        with open(variable_definition_filename, 'r') as definitions_file:
+            self.variable_definitions = json.loads(definitions_file.read())
```

Comment:
recommend not doing in IO in the constructor, will make for easier unit testing e.g. https://github.com/seattle-uat/civiform/blob/main/cloud/shared/validate_variable_definitions.py#L16-L18